client/src/dbus/service.rs

```diff
@@ -51,7 +51,7 @@ impl<'a> Service<'a> {
     pub async fn new() -> Result<Service<'a>, Error> {
         let service = match Self::encrypted().await {
             Ok(service) => Ok(service),
-            Err(Error::Zbus(zbus::Error::MethodError(_, _, _))) => Self::plain().await,
```

Comment:
I think it would be more safe to have both match branches.

Reply:
I'm fine with keeping both. Do you want me to change it in this pull request?

Reply:
i would keep both in the same commit yes, just to avoid any future regressions